tests/test_semantic_conventions.py

```diff
@@ -224,6 +224,19 @@ def validator(span):
 
         interfaces.library.add_span_validation(validator=validator)
 
+    @bug(library="cpp", reason="component tag not implemented")
+    def test_meta_runtime_id_tag(self):
+        """Assert that all spans generated from a weblog_variant have runtime-id metadata tag with some value."""
+
+        def validator(span):
+
+            if "runtime-id" not in span.get("meta"):
+                raise Exception(f"No runtime-id tag found. Expected tag to be present.")
+
+            return True
+
+        interfaces.library.add_span_validation(validator=validator)
```

Comment:
This test will validate only the very first tag it will find. If you want to test all spans, remove the return statement of `validator()`, and add the succes_by_default argument to `validate_spans` (you may need to update your branch, you are almost 200 commits behind main).

Now, on the other hand, if you only want to ensure that `runtime-id` is always present in data, regardless of its value, schemas may be a simpler solution : 

https://github.com/DataDog/system-tests/blob/main/utils/interfaces/schemas/library/v0.4/misc/trace-item.json#L17

Reply:
Hey Charles! Changed some stuff including this. Now, depending on the tag being tested, I look for either the tag to just be present in one span (such as for span.kind), or for the tag to be found on every span (such as language).